pkg/security/ebpf/probes/event_types.go

```diff
@@ -498,6 +499,19 @@ func GetSelectorsPerEventType(fentry bool) map[eval.EventType][]manager.ProbesSe
 				},
 			}},
 		},
+
+		// List of probes required to capture sysctl events
+		"sysctl": {
+			&manager.AllOf{Selectors: []manager.ProbesSelector{
```

Comment:
It seems the attachment of cgroup-based eBPF programs can fail if one already exists for a particular cgroup (https://elixir.bootlin.com/linux/v6.13.4/source/kernel/bpf/cgroup.c#L659-L664).
Should we change the selector to `BestEffort` to avoid preventing system-probe from starting up if that ever happens?

Reply:
Discussed this offline: a config parameter will be added in a follow-up PR to disable the sysctl event type in case the above issue ever occurs